setup.py

```diff
@@ -543,7 +543,7 @@ def _check_vc(self):
         # so find and add them
         if vcbase and not atlmfc_found:
             atls_lib = glob.glob(
-                vcbase + r"ATLMFC\lib\{}\atls.lib".format(self.plat_dir)
+                vcbase + r"ATLMFC\\lib\{}\\atls.lib".format(self.plat_dir)
```

Comment:
this is now wrong?

Reply:
Yes. I missed the `r` here in manual changes. I'll wait for and answer to your other comment to fix this one